test/filters/test_unsharp_mask.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+
+import kornia
+import kornia.testing as utils  # test utils
+
+import torch
+from torch.autograd import gradcheck
+from torch.testing import assert_allclose
+
+class Testunsharp:
```

Comment:
Maybe we should a numeric test, with an small predefined input tensor. @edgarriba do you think it could be interesting?

Reply:
Agree